lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,26 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # Describes car's state and behaviour
+  class Car
+    attr_accessor :year, :color, :model, :current_speed
+    def initialize(args)
+      @year = args[:year]   || 2000
+      @color = args[:color] || 'White'
+      @model = args[:model] || 'Mercedes W210'
+      @current_speed = 0
+    end
+
+    def self.default_car
+      new
+    end
+
+    def speed_up(increase_speed)
+      @current_speed += increase_speed
```

Comment:
As Alex said during the lecture, it the setter is defined, it should be used in all assignments